Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n# Datasource local storage ignored files\r\n/../../../../../:\\xampp\\htdocs\\dte-2604-1-21v-systemutvikling\\.idea/dataSources/\r\n/dataSources.local.xml\r\n# Editor-based HTTP Client requests\r\n/httpRequests/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 99d75a32ed5a9cf6634377bf075c608f5f02c48b)
+++ b/.idea/.gitignore	(date 1614086022181)
@@ -6,3 +6,4 @@
 /dataSources.local.xml
 # Editor-based HTTP Client requests
 /httpRequests/
+.idea
\ No newline at end of file
